source/optional_features/custom_branding/index.html.md.erb

```diff
@@ -19,6 +19,12 @@ customisation for the following features:
 * the background and border colour of the top banner
 * the branding of your payment confirmation email
 
+You must tell us which service you want to apply custom branding to. Custom branding applies to the test and live environments for a service.
```

Comment:
not sure if this entirely accurate -I think we have to re-apply it to live if we do it in test? and if it's in live, it won't necessarily show up in test?

Reply:
@iqbalgds is this your understanding?

Reply:
Have checked with @iqbalgds and we can only apply this at the service level.

Reply:
I think that we have to do some work to apply custom branding to live emails when we've done it with test  (we don't have to do anything on payment pages branding) - but maybe that distinction is not relevant to ours users but something we should be aware of

Reply:
agree to address this through separate ticket on internal manual to make sure gov.uk pay team members know to check